### PR TITLE
feat: use Webp for images in Eigen

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **feature**

### Description
- Use Webp for images in Eigen

Our images folder went from 11 mb to about 2.5mb. I will update this thread with the bundle size difference

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- 

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Use Webp - mounir

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Use Webp - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 87bd87f5303afe61b75d7412b1172f7433cfbb9c
</p>

File: typings/images.d.ts [hunk 1]
```diff
@@ -1 +1,2 @@
 declare module "*.png"
+declare module "*.webp"
```
Comment (MounirDhahri) [inline typings/images.d.ts:2]: this was needed to solve typescript webp imports failures

**Review by olerichter00 — APPROVED** [review]
Very nice 👍

**Comment by MounirDhahri** [comment]
# Update
## iOS

### iPhones
34.4 MB to 24.3MB (30% install size reduction)

iPads
47.5MB to 37.5 MB (21% install size reduction)

## Android

26.9 MB to 21.2MB (21% install size reduction)

**Pull request merged**